deploy/examples/csi-topology/deploy.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+# Install a Ceph demo container
+kubectl create -f 00-ceph-demo.yml
```

Comment:
Suffix needs to be .yaml, not .yml (also for all following files)

Reply:
The latest commit makes consistent usage of 'yaml' thorough out the code